html/pfappserver/root/src/views/Configuration/selfServices/_components/TheForm.vue

```diff
@@ -38,6 +38,12 @@
           :text="$i18n.t('List of OS which will be allowed to be register via the self service portal.')"
         />
       </base-form-tab>
+      <base-form-tab :title="$i18n.t('Dynamic Pre Shared Key')">
```

Comment:
```suggestion
      <base-form-tab :title="$i18n.t('DPSK')">
```

(To stay consistent with all the other places we display it like the connection profile)